packages/addons-v5/test/commands/addons/wait.js

```diff
@@ -74,6 +74,7 @@ describe('addons:wait', () => {
       })
 
       it('does NOT notify the user when provisioning takes less than 5 seconds', () => {
+        // eslint-disable-next-line node/no-extraneous-require
```

Comment:
Same question as before. This may be just nitpicks, but wanted to double check with you.